src/script.ts

```diff
@@ -0,0 +1,135 @@
+/*
+  Copyright 2019 Esri
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import { loadCss } from './utils/css';
+import utils from './utils/index';
+import { getCdnUrl } from './utils/url';
+
+function createScript(url) {
```

Comment:
i mean, its _your_ library, but it _is_ TypeScript.

```ts
function createScript(url:string) {
```

Reply:
at some point I need to rip off [this bandaid](https://github.com/Esri/esri-loader/blob/517cd90e520fd51b43e7ee872f3253f86c286004/tsconfig.json#L10), and then I can let TS tell me when I do and don't need to jump through these kind of hoops.